src/pages/Trade/Trade.tsx

```diff
@@ -21,7 +99,24 @@ export const Trade = memo(() => {
         justifyContent='center'
         gap={4}
       >
-        <MultiHopTrade isRewritingUrl />
+        <FormProvider {...methods}>
```

Comment:
Wrapped in `<FormProvider />` since this guy is now the trade tabs switch and requires it